CMakeLists.txt

```diff
@@ -106,21 +105,18 @@ SET(Boost_ADDITIONAL_VERSIONS "1.67" "1.67.0")
 
 SET(BOOST_PYTHON_MODULE_NAME "python")
 if ( USE_PYTHON_VERSIONS VERSION_GREATER 3 )
-    if ( APPLE )
-        SET( BOOST_PYTHON_MODULE_NAME ${MACOS_USE_PYTHON_MODULE} )
-    else()
-        SET( BOOST_PYTHON_MODULE_NAME "python3" )
-    endif()
+  SET( BOOST_PYTHON_MODULE_NAME ${BOOST_PYTHON_NAME} )
```

Comment:
So was the comment about Boost using a 2 digit naming convention on MacOS wrong? Or was it a temporary blip in Boost?

Reply:
I guess it depends on the distributer. For example, if you boost from homebrew and if you use boost from conda, you may see different names, even for linux/windows. So I just combined the option into one and made it configurable on all platforms.

Reply:
For homebrew on a Mac we'll still need to build 
with a -DBOOST_PYTHON_MODULE_NAME=python37 (or whatever specific version the boost_python lib is named by).

Reply:
That's right. I was hoping we would make that change in the build scripts.

Reply:
Yes, that's what I meant too.